webpack.config.js

```diff
@@ -273,6 +273,11 @@ function getWebpackConfig( { cssFilename, externalizeWordPressPackages = false }
 		plugins: _.compact( [
 			! codeSplit && new webpack.optimize.LimitChunkCountPlugin( { maxChunks: 1 } ),
 			new webpack.DefinePlugin( {
+				...( isCalypsoClient && {
+					'window.Jetpack_Static_Block_Assets_Base_Url': JSON.stringify(
+						'/calypso/images/jetpack-block-assets'
```

Comment:
Would it work not to use `JSON.stringify` here just like we don't use it for `global: 'window'` ?